sdk/servicebus/azure-servicebus/swagger/README.md

```diff
@@ -17,7 +17,7 @@ autorest --use=@autorest/python@5.0.0-preview.6
 ```
 ### Settings
 ``` yaml
-input-file: servicebus-swagger.json
+input-file: https://raw.githubusercontent.com/YijunXieMS/azure-rest-api-specs-pr/servicebus_mgmt/specification/servicebus/data-plane/servicebus-swagger.json?token=ALQFVABEGSXQX2IEVU26V2K7AYZ4K
```

Comment:
I'm trying to recall if we got it working without the token? (I remember we were unsure about that on the call where we discussed this)  I just worry that if that expires this'll break in an obscure way; but I still prefer this to the path.

If this is what it needs to work we can see how fragile it is.  I'll add it to one of my low-pri todo lists to check up on in a while to make sure this wasn't unwise.  (It does bring me back to the question we had on the call; how do the other libs manage it? I don't recall them having the token stanza?)

Reply:
Will find out why token is there. Tracking issue is https://github.com/Azure/azure-sdk-for-python/issues/12318